.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.5"
```

Comment:
let's add Python 3.4 as well, I've added it as supported version to setup.py in https://github.com/socialwifi/jsonapi-requests/commit/342c04adbfcc4f46117c3580bbb2f8e462577191